ipaserver/install/ipa_pkinit_manage.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (C) 2017  FreeIPA Contributors see COPYING for license
+#
+
+from __future__ import print_function
+
+from ipalib import api
+from ipaplatform.paths import paths
+from ipapython.admintool import AdminTool
+from ipaserver.install.krbinstance import KrbInstance, is_pkinit_enabled
+
+
+class PKINITManage(AdminTool):
+    command_name = "ipa-pkinit-manage"
+    usage = "%prog <enable|disable|status>"
+    description = "Manage PKINIT."
```

Comment:
Then maybe something like "Manage local PKINIT configuration on IPA server", but if the locality of the commands is not specified in other management tools then it is better to keep at least some semblance of consistency.